src/__internal__/label/label.style.ts

```diff
@@ -1,7 +1,7 @@
 import styled, { css } from "styled-components";
-import PropTypes from "prop-types";
+import { LabelProps } from "./label.component";
 
-const LabelStyle = styled.label`
+const LabelStyle = styled.label<Pick<LabelProps, "isRequired" | "disabled">>`
```

Comment:
I think that now will be the best time change `LabelStyle` to `StyledLabel` to match our naming convention, what do you think about that?